pkg/ocm/managedservice.go

```diff
@@ -134,6 +134,10 @@ func (c *Client) CreateManagedService(args CreateManagedServiceArgs) (*msv1.Mana
 }
 
 func (c *Client) ListManagedServices(count int) (*msv1.ManagedServiceList, error) {
+	if fedramp.Enabled() {
+		return nil, fmt.Errorf("managed services are not supported for FedRAMP clusters")
```

Comment:
@hunterkepley in the spirit of not repeating the message everywhere, how about:

```
var fedrampError = fmt.Errorf("managed services are not supported for FedRAMP clusters")
```
and then methods updated to

```
if fedramp.Enabled() {
		return nil, fedrampError
}
```

Reply:
SGTM